### PR TITLE
Followed best practises

### PR DESCRIPTION
**Pull request by subtra3t**
Certain window constants were declared with `var` instead of `const`. Now they are actually constants.

### CODE REVIEW

File: files/en-us/games/techniques/3d_on_the_web/building_up_a_basic_demo_with_three.js/index.html [hunk 1]
```diff
@@ -129,7 +129,7 @@ <h2 id="Geometry">Geometry</h2>
 
 <p>Now our scene is properly rendering, we can start adding 3D shapes. To speed up development, Three.js provides a bunch of predefined primitives, which you can use to create shapes instantly in a single line of code. There's cubes, spheres, cylinders, and more complicated shapes available. Detail like drawing required vertices and faces, for a given shape, is handled by the Three framework, so we can focus on higher level coding. Let's start, by defining the geometry for a cube shape, adding the following just above the <code>render()</code> function:</p>
 
-<pre class="brush: js">var boxGeometry = new THREE.BoxGeometry(10, 10, 10);
+<pre class="brush: js">boxGeometry = new THREE.BoxGeometry(10, 10, 10);
```
Comment (chrisdavidmills) [inline files/en-us/games/techniques/3d_on_the_web/building_up_a_basic_demo_with_three.js/index.html:132]: This is the first time `boxGeometry` is seen on the page, so this should surely have a `const` as well, not just removing the `var`?

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Thanks for the update @subtra3t ; just one comment for you to address.

File: files/en-us/games/techniques/3d_on_the_web/building_up_a_basic_demo_with_three.js/index.html (hunk 1 above)
Comment (subtra3t) [reply files/en-us/games/techniques/3d_on_the_web/building_up_a_basic_demo_with_three.js/index.html:132]: @chrisdavidmills Sorry! I also meant to declare it a constant, but it seems that all I did was just delete the `var` keyword. I will edit it out.

**Review by chrisdavidmills — APPROVED** [review]
Perfect, thank you!

**Pull request merged**